tensorrt_llm/serve/openai_server.py

```
@@ -253,6 +253,10 @@ async def create_chat_response(
                 tool.model_dump() for tool in request.tools
             ]
             sampling_params = request.to_sampling_params()
+            # TODO: better way to enable metrics
+            if len(os.getenv("TRTLLM_KVCACHE_TIME_OUTPUT_PATH", "")) > 0:
```

Ruff (0.12.2)

Undefined name `os`

(F821)